src/components/App/StepComponentMap.js

```diff
@@ -11,7 +11,7 @@ const stepToComponents = (stepDefaultOptions, {type: stepType, options: stepOpti
       return [<Select method='document' {...optionExt} />,
               <Capture method='document' {...optionExt} autoCapture={true}/>]
     case 'face':
-      return <Capture method='face' {...optionExt} autoCapture={false}/>
+      return <Capture method='face' useWebcam={true} {...optionExt} autoCapture={false}/>
```

Comment:
yeah, I just wanted to confirm if that was the intention. I think we could create two thin wrappers on `Capture` called `FaceCapture` and `DocumentCapture` with different default values to make it a bit more clear. what do you think?